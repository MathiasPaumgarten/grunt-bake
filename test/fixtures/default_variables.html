<div>
    <!--(bake-start)-->
    {{ __filename }}
    {{ __timestamp }}
    <!--(bake-end)-->
</div>