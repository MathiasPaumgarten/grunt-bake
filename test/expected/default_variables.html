<div>
    test/fixtures/default_variables.html
    123456789
</div>